components/Reference/index.tsx

```diff
@@ -112,12 +113,11 @@ const ReferenceTable = ({
   // Change selectedFork according to query param
   useEffect(() => {
     const query = router.query
-
     if ('fork' in query) {
       onForkChange(query.fork as string)
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [router.isReady])
+  }, [router.isReady && isForksLoaded])
```

Comment:
Same here

Reply:
@2xic unless you find it bothersome, may you please not mark convos as resolved and save us a few clicks 😸 . You can acknowledge by a 👍 and if you need to 👀 first until you acknowledge, you can 👀 and then remove and 👍 .

Reply:
I remember we discussed it in FE sync, not a requirement, seemed to be the consensus amongst the team back then.

Reply:
Yup, I know, but what I'm saying is that this repo (evm.codes) required us to resolve all conversations before we could merge (like some other repos we have). So even I if I wanted to follow the rules, I couldn't 😅

Anyways - I have updated the settings now so that the evm.codes repo settings matches our other FE monorepo.

Reply:
🤦 my bad, didn't notice the repo...